plugins/org.jboss.tools.livereload.ui/plugin.xml

```diff
@@ -82,9 +82,9 @@
                  <iterate
                        ifEmpty="false"
                        operator="or">
-                    	<adapt
-							type="org.eclipse.wst.server.ui.IServerModule">
-						</adapt>
+                    	<instanceof
+							value="org.eclipse.wst.server.ui.IServerModule">
+						</instanceof>
```

Comment:
@xcoulon it makes commands `org.jboss.tools.livereload.openInExternalDeviceWebBrowserViaQRCode` and `org.jboss.tools.livereload.openInWebBrowserViaLiveReloadProxy` enabled identically (see line 109). By now, without this changes, "Open in web browser via LiveReload" was enabled while "Open in External Device" was disabled, because `adapt` is a bit more than `instanceof`.